src/tests/dotnet-trace/CollectLinuxCommandFunctionalTests.cs

```diff
@@ -0,0 +1,200 @@
+                    "",
+                    LinuxHeader,
+                    LinuxProfile("cpu-sampling"),
+                    ""
+                }
+            };
+            yield return new object[] {
+                TestArgs(clrEvents: "gc", profiles: new[]{"cpu-sampling"}),
+                new string[] {
+                    "", ProviderHeader,
+                    FormatProvider("Microsoft-Windows-DotNETRuntime","0000000000000001","Informational",4,"--clrevents"),
+                    "",
+                    LinuxHeader,
+                    LinuxProfile("cpu-sampling"),
+                    ""
+                }
+            };
+            yield return new object[] {
+                TestArgs(providers: new[]{"Microsoft-Windows-DotNETRuntime:0x1:4"}, profiles: new[]{"cpu-sampling"}),
+                new string[] {
+                    "", ProviderHeader,
+                    FormatProvider("Microsoft-Windows-DotNETRuntime","0000000000000001","Informational",4,"--providers"),
+                    "",
+                    LinuxHeader,
+                    LinuxProfile("cpu-sampling"),
+                    ""
+                }
+            };
+            yield return new object[] {
+                TestArgs(providers: new[]{"Microsoft-Windows-DotNETRuntime:0x1:4"}, clrEvents: "gc"),
+                new string[] {
+                    "Warning: The CLR provider was already specified through --providers or --profile. Ignoring --clrevents.",
+                    "", ProviderHeader,
+                    FormatProvider("Microsoft-Windows-DotNETRuntime","0000000000000001","Informational",4,"--providers"),
+                    "",
+                    LinuxHeader,
+                    ""
+                }
+            };
+            yield return new object[] {
+                TestArgs(clrEvents: "gc+jit"),
+                new string[] {
+                    "", ProviderHeader,
+                    FormatProvider("Microsoft-Windows-DotNETRuntime","0000000000000011","Informational",4,"--clrevents"),
+                    "",
+                    LinuxHeader,
+                    ""
+                }
+            };
+            yield return new object[] {
+                TestArgs(clrEvents: "gc+jit", clrEventLevel: "5"),
+                new string[] {
+                    "", ProviderHeader,
+                    FormatProvider("Microsoft-Windows-DotNETRuntime","0000000000000011","Verbose",5,"--clrevents"),
+                    "",
+                    LinuxHeader,
+                    ""
+                }
+            };
+            yield return new object[] {
+                TestArgs(perfEvents: new[]{"sched:sched_switch"}),
+                new string[] {
+                    "No .NET providers were configured.",
+                    "",
+                    LinuxHeader,
+                    LinuxPerfEvent("sched:sched_switch"),
+                    ""
+                }
+            };
```

Comment:
What happens if the user specifies a profile that is only supported by 'collect'? I think that error case would make a good test case.